app/models.py

```diff
@@ -965,7 +965,9 @@ def clarification_questions_are_closed(self):
 
     @hybrid_property
     def status(self):
-        if self.published_at is None:
+        if self.withdrawn_at:
```

Comment:
Any reason you're using `withdrawn_at`'s truthiness here instead of `is None` as is used against `published_at`?